plugins/fluent-bit/src/plugin.cpp

```diff
@@ -130,6 +130,8 @@ auto visit(auto f, const msgpack_object& object) {
       return f(to_bin(object));
     case MSGPACK_OBJECT_EXT:
       return f(object.via.ext);
+    default:
+      return f(ec::unspecified);
```

Comment:
Having a `default:` label here, instead of having this outside the `switch`, has the effect that the compiler won't warn about any potentially missing enumerators. This enum isn't probably going to get new enumerators any time soon, but that's something to consider.